be/src/exec/olap_scan_node.cpp

```diff
@@ -702,8 +694,13 @@ Status OlapScanNode::start_scan_thread(RuntimeState* state) {
                 state, this, _olap_scan_node.is_preaggregation, _need_agg_finalize, *scan_range, scanner_ranges);
             _scanner_pool->add(scanner);
             _olap_scanners.push_back(scanner);
+
+            if (disk_set.find(scanner->scan_disk()) == disk_set.end()) {
+                disk_set.insert(scanner->scan_disk());
+            }
         }
     }
+    COUNTER_SET(_num_disks_accessed_counter, static_cast<int64_t>(disk_set.size()));
```

Comment:
What is this metric used for? If this metric is not useful, just remove it

Reply:
The metric help we now how many disk accessed in olap scan node scan data

Reply:
What can it be used for when you know how many disks accessed?

Reply:
We can know whether the tablet of a query is allocated on different disks to know whether is IO bottleneck in disk